mlflow/server/__init__.py

```diff
@@ -7,9 +7,11 @@
 from mlflow.server.handlers import get_artifact_handler
 from mlflow.utils.process import exec_cmd
 
-BACKEND_STORE_URI_ENV_VAR = "MLFLOW_SERVER_FILE_STORE"
-ARTIFACT_ROOT_ENV_VAR = "MLFLOW_SERVER_ARTIFACT_ROOT"
-STATIC_PREFIX_ENV_VAR = "MLFLOW_STATIC_PREFIX"
+# NB: These are intenrnal environment variables used for communication between
+# the cli and the forked gunicorn processes.
+BACKEND_STORE_URI_ENV_VAR = "_MLFLOW_SERVER_FILE_STORE"
```

Comment:
These are not the best environment variable names, but they are not user-facing and have no bearing other than being unique, so I just prefixed them with "_" to help indicate privacy.